lib/positioning/healer.rb

```diff
@@ -8,7 +8,7 @@ def initialize(model, column, order)
 
     def heal
       if positioning_columns.present?
-        @model.select(*positioning_columns).distinct.each do |scope_record|
+        @model.unscoped.select(*positioning_columns).distinct.each do |scope_record|
```

Comment:
This might be too aggressive? It is possible to just `.unscope(:order)` to remove the order. I'm erring toward that at the moment and perhaps if other types of default scoping cause issues then those people can raise issues for that.

Reply:
Sounds good. I didn't realize there was `#unscope(value)` in addition to `#unscoped`, but it makes total sense to use the narrower variant.

Reply:
Indeed :) It is pretty handy :D